### DISCUSSION TITLE
143

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#79] [General]
# 潮流周刊第143期 - 粉色的云

封面图来源于上周刚好在情人节那一天回家路上随手拍，云居然还粉色的，是懂那么一点浪漫的云。 

https://weekly.tw93.fun/posts/143-%E7%B2%89%E8%89%B2%E7%9A%84%E4%BA%91

**Comment by davidchen27** [comment]
你好, 想请教下, 有没有关于本站字体加载方案的介绍文章之类的, 对此感兴趣😊

**Reply by tw93** [reply]
可以参考这个  https://github.com/chawyehsu/lxgw-wenkai-webfont

**Reply by davidchen27** [reply]
感谢~

**Comment by Qusunyee** [comment]
👍👍

**Comment by QuakeWang** [comment]
Wow！发现自己和 tw93 大佬工作的地方离的好近，那天下班也看见粉色的云叻～～（未来科技城。

**Reply by tw93** [reply]
哈哈哈  握手握手
